bool is_people_follow